tests/bud/multiarch/Dockerfile.no-run

```diff
@@ -3,5 +3,5 @@ FROM docker.io/library/alpine
 COPY Dockerfile.no-run /root/
 # A different base image that is known to be a manifest list, supporting a
 # different but partially-overlapping set of platforms.
-FROM registry.access.redhat.com/ubi8-micro
+FROM quay.io/libpod/testimage:20221018
```

Comment:
FIXME FIXME FIXME FIXME!!! 

I can't use `$SAFEIMAGE` here. No matter where I put `ARG SAFEIMAGE` (line 1 at top, or line 5 just above the `FROM`), buildah barfs with:
```
Error: resolving image name "$SAFEIMAGE": cannot parse input: "$SAFEIMAGE": invalid reference format
```
But it works perfectly fine in the other test. Help please.

Reply:
UPDATE: the culprit is `--all-platforms`. It looks like `--all-platforms` cannot be combined with `--build-args`. Sounds like a bug to me, should I file an issue? Or is this a yeah-deal-with-it thing?

Reply:
@flouthoc PTAL